Makefile

```diff
@@ -65,5 +66,8 @@ docker-build: test
 docker-push:
 	docker push ${IMG}
 
+depend:
+	dep ensure
```

Comment:
I think you need some logic as https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/Makefile#L45-L47